test/suites/rt-upgrade/test-upgrade-chain.ts

```diff
@@ -0,0 +1,63 @@
+import { describeSuite, beforeAll, expect } from "@moonwall/cli";
+import { alith } from "@moonwall/util";
+import { ApiPromise } from "@polkadot/api";
+import { parseEther } from "ethers";
+import "@moonbeam-network/api-augment";
+
+describeSuite({
+  id: "CHU",
+  title: "Chopsticks Upgrade",
+  foundationMethods: "chopsticks",
+  testCases: ({ it, context }) => {
+    let api: ApiPromise;
+    const DUMMY_ACCOUNT = "0x11d88f59425cbc1867883fcf93614bf70e87E854";
+
+    beforeAll(() => {
+      api = context.polkadotJs();
+    });
+
+    it({
+      id: "T1",
+      title: "Calling chain constants data",
+      test: async () => {
+        const specVersion = api.consts.system.version.specVersion.toNumber();
```

Comment:
Not sure what this tests

Reply:
Sanity check to make sure that:
1. Chopsticks is actually connected to *something*, and that it hasn't failed silently
2. Our polkadotJs provider is actually connected to a chain

I'll change this to look at the runtime name, as a less trite test.

Reply:
I'd include that in the before and not being at test as itself, as it seems to be more a requirement than verifying a behavior

Reply:
Done